agent/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
 
     testImplementation "org.junit.jupiter:junit-jupiter-api:$junitVersion"
+    testImplementation 'org.mockito:mockito-core:5.20.0'
```

Comment:
As this dependency is also used in the model package,
I would use the variable approach and specify its version in the `gradle.properties` file.